CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+# Contributing to parity-common
+
+parity-common welcomes contribution from everyone in the form of suggestions, bug
+reports, pull requests, and feedback. This document gives some guidance if you
+are thinking of helping us.
+
+Please reach out here in a GitHub issue if we can do anything to help you contribute.
```

Comment:
mention parity gitter/matrix?

Reply:
I think we're using parity gitter as parity-ethereum support channel. Here the users of parity-common are rust developers exclusively and ATM we don't have many questions coming up about crates in parity-common. If we will, I would suggest creating a separate channel for that.

Reply:
We're using both (I think there's a bridge too?). I think we should mention at least the gitter.